src/CommandHook.hpp

```diff
@@ -0,0 +1,96 @@
+#ifndef __COMMAND_HOOK_HPP__
+#define __COMMAND_HOOK_HPP__
+
+#include <string>
+
+#include <mesos/hook.hpp>
+#include <mesos/module/hook.hpp>
+
+namespace criteo {
+namespace mesos {
+
+class CommandHook : public ::mesos::Hook
+{
+public:
+  /*
+   * Constructor
+   * Each argument can be empty. In that case the corresponding hook
+   * method will not be treated.
+   *
+   * @param runTaskLabelCommand The command used by
+   * slaveRunTaskLabelDecorator method if provided.
+   * @param executorEnvironmentCommand The command used by
+   * slaveExecutorEnvironmentDecorator if provided.
+   * @param removeExecutorCommand The command used by
+   * slaveRemoveExecutorHook if provided.
+   */
+  CommandHook(const std::string& runTaskLabelCommand,
```

Comment:
Could have used a map<string,string> here ... easier to inspect, validate, etc

Reply:
FAR